<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>图形</string>
    </property>
    <addaction name="act_line"/>
    <addaction name="act_ecl"/>
    <addaction name="act_rect"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="act_sel"/>
    <addaction name="act_redo"/>
    <addaction name="act_undo"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="act_open"/>
    <addaction name="act_save"/>
   </widget>
   <addaction name="menu_3"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="act_line"/>
   <addaction name="act_rect"/>
   <addaction name="act_ecl"/>
   <addaction name="separator"/>
   <addaction name="act_sel"/>
   <addaction name="act_undo"/>
   <addaction name="act_redo"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="act_line">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/img/res/line.bmp</normaloff>:/new/img/res/line.bmp</iconset>
   </property>
   <property name="text">
    <string>直线</string>
   </property>
  </action>
  <action name="act_ecl">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/img/res/ellc.bmp</normaloff>:/new/img/res/ellc.bmp</iconset>
   </property>
   <property name="text">
    <string>椭圆</string>
   </property>
  </action>
  <action name="act_rect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/img/res/rect.bmp</normaloff>:/new/img/res/rect.bmp</iconset>
   </property>
   <property name="text">
    <string>矩形</string>
   </property>
  </action>
  <action name="act_sel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>选择</string>
   </property>
  </action>
  <action name="act_redo">
   <property name="text">
    <string>重做</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="act_undo">
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="act_open">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="act_save">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
